src/index.tsx

```diff
@@ -1,73 +1,89 @@
-import * as React from "react";
-import * as ReactDOM from "react-dom";
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
 
-import * as XC from 'trc-httpshim/xclient'
-import * as common from 'trc-httpshim/common'
-import * as core from 'trc-core/core'
-import * as trcSheet from 'trc-sheet/sheet'
+import { SheetContainer } from 'trc-react/dist/SheetContainer';
 
-import { SheetContainer, IMajorState } from 'trc-react/dist/SheetContainer'
-import { VrdbLookup, IVoter } from './components/VrdbLookup'
+import { Copy } from 'trc-react/dist/common/Copy';
+import { CsvMatchInput } from 'trc-react/dist/CsvMatchInput';
+import { ListColumns } from 'trc-react/dist/ListColumns';
+import { Panel } from 'trc-react/dist/common/Panel';
+import { PluginShell } from 'trc-react/dist/PluginShell';
 
-import * as bcl from 'trc-analyze/collections'
-import { PluginLink } from "trc-react/dist/PluginLink";
-import { ColumnNames } from "trc-sheet/sheetContents";
+// Lets somebody lookup a voter, and then answer questions about them.
+// See all answers in Audit.
 
+interface IVoter {
```

Comment:
IS this being used? May be able to delete (may have been copied from another plugin)

Reply:
Yes, the `IVoter` interface is used for the `voter` property in the state of the `<App>` component.